app/config/extensions/console.neon

```diff
@@ -1,5 +1,8 @@
 extensions:
-    - Contributte\Console\DI\ConsoleExtension
+    console: Contributte\Console\DI\ConsoleExtension
+
+console:
+    url: https://eventigo.cz
```

Comment:
@f3l1x Thanks for the PR 👍  

Could we use this parameter: https://github.com/eventigo/eventigo-web/blob/master/app/config/parameters.neon#L2 ?